mlrun/api/db/sqldb/models/models_mysql.py

```diff
@@ -94,6 +94,25 @@ def get_identifier_string(self) -> str:
     return Tag
 
 
+def make_artifact_tag(table):
```

Comment:
Add a comment explaining why we need this and cannot use the generic `make_tag_v2` (which is the usage of `obj_id` rather than `object_name` IIRC).